src/azure-cli/azure/cli/command_modules/appservice/_help.py

```diff
@@ -1715,6 +1715,35 @@
         az webapp deployment user show
 """
 
+helps['webapp deployment github-actions'] = """
+type: group
+short-summary: Configure GitHub Actions for a webapp
+"""
+
+helps['webapp deployment github-actions add'] = """
+    type: command
+    short-summary: Add a GitHub Actions workflow file to the specified repository. The workflow will build and deploy your app to the specified webapp.
+    examples:
+      - name: Add GitHub Actions to a specified repository, providing personal access token
+        text: >
+            az webapp deployment github-actions add --repo "githubUser/githubRepo" -g MyResourceGroup -n MyWebapp --token MyPersonalAccessToken
+      - name: Add GitHub Actions to a specified repository, using interactive method of retrieving personal access token
+        text: >
+            az webapp deployment github-actions add --repo "githubUser/githubRepo" -g MyResourceGroup -n MyWebapp --login-with-github
```

Comment:
it would be better if you can remove the spaces here to align with other help entries